content/api/hub/sample_http_auth_middleware.md

```diff
@@ -0,0 +1,63 @@
+title = "HTTP Auth Middleware"
+template = "render_hub_content_body"
+date = "2023-11-03T00:00:00Z"
+content-type = "text/html"
+tags = ["rust", "authentication", "http"]
+
+[extra]
+author = "fermyon"
+type = "hub_document"
+category = "Sample"
+language = "rust"
+created_at = "2023-11-03T00:00:00Z"
+last_updated = "2023-11-07T00:00:00Z"
+spin_version = ">=v2.0"
+summary =  "Example of how to compose a middleware component with a business logic component"
+url = "https://github.com/fermyon/http-auth-middleware"
+keywords = "rust, http, auth"
+
+---
+
+This sample shows how to compose a middleware component with a business logic component using [Spin 2.0](https://www.fermyon.com/blog/introducing-spin-v2) and the Component Model. 
+
+## Instructions
+
+### Install Prerequisites
+* Install the [cargo component](https://github.com/bytecodealliance/cargo-component):
+
+```bash
+cargo install --git https://github.com/bytecodealliance/cargo-component cargo-component
+```
+
+* Install a fork of [wasm-tools]():
+
+```bash
+cargo install --git https://github.com/dicej/wasm-tools --branch wasm-compose-resource-imports wasm-tools --locked
+```
+
+* Install latest [Spin](https://github.com/fermyon/spin)
```

Comment:
Why not direct them to the Install page in the docs?

Reply:
Good suggestion. I've modified the URL.